frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -193,6 +195,33 @@ class WorkspaceAdvanced extends React.Component {
     }
   }
 
+  handleToggleCalendarEnabled = async () => {
+    const { props, state } = this
+    const oldCalendarEnabledValue = state.content.calendar_enabled
+    const newCalendarEnabledValue = !state.content.calendar_enabled
+
+    this.setState(prev => ({content: {...prev.content, calendar_enabled: newCalendarEnabledValue}}))
+    const fetchToggleCalendarEnabled = await handleFetchResult(await putCalendarEnabled(state.config.apiUrl, state.content, newCalendarEnabledValue))
+
+    switch (fetchToggleCalendarEnabled.apiResponse.status) {
+      case 200:
+        this.sendGlobalFlashMessage(
+          newCalendarEnabledValue ? props.t('Calendar activated') : props.t('Calendar deactivated'),
+          'info'
+        )
+        GLOBAL_dispatchEvent({ type: 'refreshWorkspaceList', data: {} }) // for sidebar and dashboard and admin workspace
```

Comment:
missing `type`, `date` and `name` for comment

Reply:
to delete comment

Reply:
comment fixed